settings/chrome-dev.json

```diff
@@ -8,5 +8,6 @@
 
   "googleAnalytics": "UA-26026798-6",
 
-  "browserIsChrome": true
+  "browserIsChrome": true,
```

Comment:
I wondered where `browserIsChrome` is used at first, then I saw that it is used by the extension manifest template. In future we might want to separate out the settings that are only used when building the extension vs. those that are used by the client.